runtime/rastrace/trcengine.c

```diff
@@ -635,17 +634,17 @@ initializeTraceOptions(J9JavaVM *vm, char* opts[])
 	IDATA rc =  J9VMDLLMAIN_OK;
 
 	/* set up the default options */
-	rc = splitCommandLineOption(vm, UT_MAXIMAL_KEYWORD "=all{level1}", 20, opts);
+	rc = splitCommandLineOption(vm, UT_MAXIMAL_KEYWORD "=all{level1}", 19, opts);
```

Comment:
Why are these numbers  changing?

Reply:
This code (from days gone by) is particularly idiotic, in that it uses a macro for some of the string, so there's no way to accurately predict the length. Given there's only a few uses of constant strings, perhaps it would be better to add a new function that strlen's the input and calls the general one.

Reply:
The numbers are being corrected; perhaps some part used to be longer and the length didn't get updated.

I will add a macro that can use `sizeof` to compute the length of the string literals.

Reply:
I don't think that will work - sizeof will include the trailing nul character. strlen of a constant string should be folded by the compiler.

Reply:
That's ok, I can subtract one. ;-)

Reply:
IIRC, we had issues on some compilers before, where sizeof wasn't consistent on strings. Given that the code clearly doesn't need the correct number, I suppose being off by one won't be terrible.